rotkehlchen/chain/evm/transactions.py

```diff
@@ -294,10 +302,21 @@ def _query_and_save_internal_transactions_for_range_or_parent_hash(
         If update_ranges is True, updates the database tracking for this query range.
         Otherwise, data is fetched without updating the query range.
         """
+        query_period_or_hash = period_or_hash
```

Comment:
This is same as in line 198. Can/should it be abstracted or does not make sense since it's just 2?